docs/topics/kubernetes-developers.md

```diff
@@ -43,4 +43,18 @@ For VERSION, we recommend that you provide a value which would help you identify
 }
 ```
 
+* AKS Engine uses the "aks-ubuntu-1604" image for the master and linux agents. This image contains prebuilt versions of kubelet and kubectl, thus the provision scripts will not copy the desired kubelet binary ( and kubectl ) from the custom hyperkube image. In order to get over this limitation it is required to specify "ubuntu" ( or any other distro ) for the master and linux agents in "masterProfile" and each agent pool under "agentPoolProfiles". 

```

Comment:
I would remove the "or any other distro" part since we have different versions of the aks distro (moby, 18.04, etc)

Reply:
Fair enough. Done.